tests/test_definitions.py

```diff
@@ -92,12 +104,23 @@ def test_convert_magicc7_to_openscm_variables(magicc7, openscm):
     [
         ("HFC245FA_EMIS", "Emissions|HFC245fa"),
         ("CO2I_EMIS", "Emissions|CO2|MAGICC Fossil and Industrial"),
-        ("CH4_EMIS", "Emissions|CH4"),
         ("CH3CCL3_EMIS", "Emissions|CH3CCl3"),
         ("CCL4_INVERSE_EMIS", "Inverse Emissions|CCl4"),
+        ("CO2_EMIS", "Emissions|CO2"),
```

Comment:
Should this be CO2T_EMIS? In the source there is no reference to a CO2_EMIS variable

Reply:
I think it doesn't actually matter as MAGICC has to have a fossil/afolu split (i.e. you could never prescribe total CO2 emissions). In any case, I have followed the convention for methane. In the output, `CH4T_EMIS` is total methane emissions but in the input of a SCEN7 file, the column header has to be `CH4_EMIS`. Hence the mapping has to be different depending on whether you're going forwards or backwards.